test/tests/functional/pbs_calendaring.py

```diff
@@ -64,20 +64,20 @@ def test_topjob_start_time(self):
         self.server.manager(MGR_CMD_SET, SCHED, a)
 
         res_req = {'Resource_List.select': '1:ncpus=1',
-                   'Resource_List.walltime': 10,
+                   'Resource_List.walltime': 30,
                    'array_indices_submitted': '1-6'}
         j1 = Job(TEST_USER, attrs=res_req)
-        j1.set_sleep_time(10)
+        j1.set_sleep_time(30)
         jid1 = self.server.submit(j1)
         j1_sub1 = j1.create_subjob_id(jid1, 1)
         j1_sub2 = j1.create_subjob_id(jid1, 2)
 
         res_req = {'Resource_List.select': '1:ncpus=1',
-                   'Resource_List.walltime': 10}
+                   'Resource_List.walltime': 30}
         j2 = Job(TEST_USER, attrs=res_req)
         jid2 = self.server.submit(j2)
 
-        self.server.expect(JOB, {'job_state': 'X'}, j1_sub1)
+        self.server.expect(JOB, {'job_state': 'X'}, j1_sub1, interval=1)
```

Comment:
I can add a check for a parent job to be in 'B' state but I think that will not help. 
So what happens is that at time X first subjob starts running. We expect the first subjob to end and the second subjob to run. At this point, we check the EST of array job and normal job submitted after that. If everything goes right we will find EST of array job at X+20 and the normal job at X+30. Unfortunately on slow machines that do not happen.

By the time we do the status of array job to find EST another subjob ends but we see that there is no update to EST of the jobs but when we do the status of second normal job EST of both array job and normal jobs have been updated. Which means we get EST of array parent as X+10 and EST of the normal job as X+30 this is why the test fails.

So, I thought it would just be better that we keep the job so far apart that even on slower systems this would not happen.